core/css.js

```diff
@@ -136,7 +136,6 @@ Blockly.Css.CONTENT = [
 
   '.blocklyNonSelectable {',
     'user-select: none;',
-    '-moz-user-select: none;',
```

Comment:
FYI: The current version of Mozilla now supports `user-select` without the `-moz` prefix.  And previous versions of Mozilla started supporting `-webkit-user-select` (!).  So these lines aren't needed any more.

Reply:
Should we hold off on this one until the next release? Firefox 68 still has 4% market share in September 2019 according to https://www.stetic.com/market-share/browser/

Reply:
Public support in Firefox for `-webkit-user-select` (which we are keeping) was added in version 49.  That's from 2016.
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

Additionally, the failure mode is benign, it's just the ability to select text that probably shouldn't be selectable.

Reply:
Sounds good.